aspnetcore/signalr/security.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Security Considerations in ASP.NET Core SignalR
+author: rachelappel
+description: Learn how to use authentication and authorization in ASP.NET Core SignalR.
+monikerRange: '>= aspnetcore-2.1'
+ms.author: rachelap
+ms.custom: mvc
+ms.date: 06/29/2018
+uid: signalr/security
+---
+
+# Security Concepts in ASP.NET Core SignalR
```

Comment:
Do you prefer the word "concepts" or "considerations" here? The title metadata above should match.

Reply:
Derp. Yeah I like considerations, I'll make them consistent.